client/src/components/container/admin/LogInPage/index.js

```diff
@@ -0,0 +1,88 @@
+import React, {Component} from 'react';
+import axios from 'axios';
+import { Button ,Form,Input, Alert,message} from 'antd';
+import {MailOutlined, LockOutlined} from '@ant-design/icons';
+
+import './style.css'
+import  Logo from './images/logo.svg';
+import adminLoginImg from './images/adminlogin.png';
+
+class AdminLogin extends Component {
+    state ={
+        error:'',
+        isLoaded: false,
+    }
+
+    onFinish=({email, password})=>{
+        this.setState({isLoaded:true})
+
+
+    axios.post(`/api/v1/login`, {email,password}).then(({data})=>{
+        if(data.status === 400){
+            this.setState({isLoaded:false})
+           this.setState({error:data.msg})
+        }
+         else if (data.status === 401){
+            this.setState({error:data.msg})
+        } else{
+            const {history:{push}} = this.props;
+            push('/admin/dashboard')
```

Comment:
here, use `antd` message  to send a notification with data.msg that will show "logged successfully" : 
```suggestion
            const {history:{push}} = this.props;
            message.success(data.msg)
            push('/admin/dashboard')
```